video/out/vo_kitty.c

```diff
@@ -114,6 +121,35 @@ static struct sigaction saved_sigaction = {0};
 static bool resized;
 #endif
 
+static inline void write_bstr_passthrough(struct priv *p, bstr bs)
+{
+    write_bstr(p->dcs_prefix);
+    write_bstr(bs);
+    write_bstr(p->dcs_suffix);
+}
+
+static inline void bstr_xappend_passthrough(struct priv *p, bstr *bs,
```

Comment:
I would name those functions in less `bstr` way. Simple `append` and `append_asprintf` are be fine. But what you prefer.